engine/src/main/java/org/terasology/config/flexible/validators/RangedNumberValidator.java

```diff
@@ -78,4 +82,10 @@ public boolean validate(T value) {
 
         return withinMinBoundary && withinMaxBoundary;
     }
+
+    @Override
+    public void issueWarnings(T value) {
+        LOGGER.warn("Value {} is not in the range {}{}, {}{}", value, minInclusive ? "[" : "(",
+                min != null ? min : "UNBOUNDED", max != null ? max : "UNBOUNDED", maxInclusive ? "]" : ")");
```

Comment:
Hmmm. I see what you mean. 

Could we provide the validator instance with the settings's SimpleURI? I.e. in a validator.setSettingURI(SimpleUri) to be called just once in the setting's constructor? I guess this prevents validator sharing between settings, but is that a probable scenario? I'm guessing not.